site/docs/get-started/docker.md

```diff
@@ -10,6 +10,12 @@ is available).
 
 {% assign release = site.data.releases["stable"] %}
 
+Before you start, make sure the Docker service is running:
+
+```bash
+$ sudo service docker start
```

Comment:
This is only on Linux, can you clarify that or maybe just leave off the command? (I think it's reasonable to expect that someone about to build a Docker image knows how to start it, or is using a GUI)